content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -2,15 +2,15 @@
 
 You can add models from any ML project to the model registry. To add a model to
 your model registry, Iterative Studio creates an annotation for it in a
-`dvc.yaml` file in your Git repository. If you are using the [GTO] command line
-tool, you can also add models [from the CLI][gto annotate]. To add models using
-Iterative Studio, watch this tutorial video or read on below:
+`dvc.yaml` file in your Git repository. You can also use [dvclive] to log your
+model during the training process or edit `dvc.yaml` directly. To add models
+using Iterative Studio, watch this tutorial video or read on below:
```

Comment:
It would be nice to explain all of these options in more detail, but one quick fix would be to call them out here more clearly like this:

```suggestion
`dvc.yaml` file in your Git repository. You can add a model in any of the following ways:

1. Log your model during the training process using [dvclive].
2. Edit `dvc.yaml` directly.
3. Use the Iterative Studio interface (watch this tutorial video or read on below).
```